src/hooks/custom-hooks/useIsP2PBlocked.ts

```diff
@@ -16,7 +16,6 @@ const useIsP2PBlocked = () => {
         if (!accountStatus) return '';
 
         if (accountStatus?.cashier_validation?.includes('system_maintenance')) return 'systemMaintenance';
-        if (accountStatus.status.includes('cashier_locked')) return 'cashierLocked';
```

Comment:
how will this affect the existing clients?

Reply:
if we remove cashier locked, we still fall back to the main blocked page because if the advertiser has cashier_locked status applied in BO, once they return to p2p, p2p_advertiser_info returns PermissionDenied error code, and we show the blocked screen